src/cljs/rems/administration/catalogue_item.cljs

```diff
@@ -70,6 +71,11 @@
                         [atoms/link nil
                          (str "/administration/forms/" (:formid catalogue-item))
                          (:form-name catalogue-item)])]
+                     [inline-info-field (text :t.administration/categories)
+                      (when-let [categories (:categories catalogue-item)]
+                        (into [:<>]
+                              (->> (map #(localized (:category/title %)) categories)
+                                   (interpose [:br]))))]
```

Comment:
Or a comma separated list?

Reply:
can do that too! i saw this line break separated list elsewhere and figured to use the same pattern.

for a horizontal list i would almost like to implement a "tag" component that separates the categories visually.. but it would probably look too different from other components and we would need to re-think about other read-only list components as well then.

Reply:
I guess it depends on the actualy values whether it should be a regular text-like list with comma separation, one row per item / <ul><li> ... or even a small table / group. I think these categories are perhaps long enough to have one row for each though there'd mostly be just one value. So not exactly like tags but I can see uses for both of these styles as common components in the future. Could be a refactor idea ticket.